container.go

```diff
@@ -25,6 +25,7 @@ type ContainerProvider interface {
 
 // Container allows getting info about and controlling a single container instance
 type Container interface {
+	GetContainerID(context.Context) string                          // get the container id from the provider
```

Comment:
This method can't be async (since the id is static and retrieving at container's startup), is it really necessary to pass the context as an argument?

Reply:
I have removed it, it should not be a requirement